src/TrieUtils.js

```diff
@@ -57,7 +57,12 @@ export function ensureSize(iter) {
 }
 
 export function wrapIndex(iter, index) {
-  return index >= 0 ? (+index) : ensureSize(iter) + (+index);
+  if (index === '' || isNaN(+index)) {
```

Comment:
We actually want to implement (as best we can without being breaking) the ECMA rules for detecting and coercing of "array index".